client/src/pages/CreateLesson/CreateLesson.js

```diff
@@ -17,6 +17,8 @@ const CreateLesson = ({ navigation }) => {
   const [name, setName] = useState('') // the name of the lesson
   const [purpose, setPurpose] = useState('') // the purpose/description of the lesson
 
+  const areAllFilled = name !== '' && purpose !== '';
```

Comment:
It would be nice to see a comment explaining the purpose of `areAllFilled`. Why do we need it?